src/checksum/zseries/crc32-s390x.c

```diff
@@ -33,8 +33,7 @@ unsigned int __wt_crc32c_le_vgfm_16(unsigned int crc, const unsigned char *buf,
  * __wt_crc32c_le --
  *     Pure C implementations of CRC, one byte at a time
  */
-unsigned int
-__wt_crc32c_le(unsigned int crc, const unsigned char *buf, size_t len)
+unsigned int __wt_crc32c_le(unsigned int crc, const unsigned char *buf, size_t len)
```

Comment:
This is to avoid "dist/s_funcs" to pick up this function (a false positive). Also, keeping it in one-line is consistent with other function definitions in this source file.

Reply:
Does adding `__wt_crc32c_le` to `s_funcs.list` work here? It seems like a smaller change than no longer auto-formatting this file.

Reply:
I was a bit on the fence about this one. Yes, it'll work adding the function into the skip list (instead of the whole file) trading off consistency, but I agree that's a smaller change. I'm happy to update it as suggested.

Reply:
See [c629fc8](https://github.com/wiredtiger/wiredtiger/pull/8360/commits/c629fc8fc3b9482e5b8e40947b3086b7edc0fbd7)